### ISSUE TITLE
Add web browser testing using Screenplay

### ISSUE THREAD

**Issue by craigfowler** [csf-dev/ZPT-Sharp#223]
Once **[Screenplay]** is ready for prime-time, it will be possible to configure Travis & Sauce Labs to perform automated tests on ZPT-Sharp renderings using a real web browser.

This will then allow full system testing of an MVC application using ZPT-Sharp, in a real application environment.

[Screenplay]:https://github.com/csf-dev/CSF.Screenplay

**Comment by craigfowler** [comment]
Blocked on Screenplay getting to a first stable release.

**Comment by craigfowler** [comment]
I'm actually going to close this as a WONTFIX.  Sure it would be a way of testing Screenplay, but would there be any benefit here, beyond the existing integration tests?  The only thing I might be testing is that the webserver comes up and can render one page successfully.  Beyond that there's no benefit beyond the existing integration tests.
